content/how-to/cloud-servers/configure-mysql-source-replica-replication/index.md

```diff
@@ -0,0 +1,310 @@
+---
+permalink: configure-mysql-source-replica-replication/
+audit_date:
```

Comment:
Don't we usually add an audit date to new articles, since they get the same verification that articles being audited get?

Reply:
doh!